toontown/leveleditor/AutoSaver.py

```diff
@@ -14,6 +14,10 @@ class AutoSaver:
     @staticmethod
     def initializeAutoSaver():
         threading.Thread(target = AutoSaver.autoSaverProcess, daemon = True).start()
+        # Creates 'autosaves' directory in user data directory
+        if not os.path.isdir('leveleditor/autosaves'):
```

Comment:
Please don't hardcode the 'leveleditor' folder as the userdata folder, theres a builtin userfiles variable, so just do f'{userdata}/autosaves'

Reply:
nvm i can just fix it myself, such a small change to delay this pr for